### PR TITLE
Implemented new layout for activity post

### PR DESCRIPTION
**Pull request by dwayhs**
screen shot 2015-10-07 at 10 22 59 am

### CODE REVIEW

File: app/assets/stylesheets/base/forms.scss [hunk 1]
```diff
@@ -1 +1,5 @@
 @charset 'utf-8';
+
+input[type="text"] {
```
Comment (css-guy) [inline app/assets/stylesheets/base/forms.scss]: Não poderia ser uma classe?

File: app/assets/stylesheets/components/activity.scss [hunk 2]
```diff
@@ -1,5 +1,47 @@
 @charset 'utf-8';
 
+.activity-form {
+  @include clearfix;
+  border-bottom: 1px solid $color-background;
+  margin-left: -$spacing;
+  margin-right: -$spacing;
```
Comment (css-guy) [inline app/assets/stylesheets/components/activity.scss]: Pode ser declarado com "()" para isolar o operador: margin: 0 (-$spacing);

**Comment by johnvoloski** [comment]
:+1:

File: app/assets/stylesheets/components/activity.scss [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+    height: 34px;
+    padding: 0 6px;
+    width: calc(100% - 35px * 2);
+  }
+
+  &__button {
+    background-color: $color-background-card;
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: 14px 14px;
```
Comment (css-guy) [inline app/assets/stylesheets/components/activity.scss]: Poderia ser: background: $color-background-card center/14px no-repeat;

File: app/assets/stylesheets/base/forms.scss (hunk 1 above)
Comment (dwayhs) [inline app/assets/stylesheets/base/forms.scss]: I'm not using a class cause it's a global reset.

I'm thinking about removing normalize to create a custom reset css.

File: app/assets/stylesheets/components/activity.scss (hunk 2 above)
Comment (dwayhs) [inline app/assets/stylesheets/components/activity.scss]: :+1: Nice!

File: app/assets/stylesheets/components/activity.scss (hunk 3 above)
Comment (dwayhs) [inline app/assets/stylesheets/components/activity.scss]: :+1:

**Pull request merged**